src/supersid_sampler.py

```diff
@@ -859,17 +858,17 @@ def doTest(args, device, sampling_rate, format):
         if 'alsaaudio' in audio_modules:
             devices = alsaaudio.pcms()
             for device in devices:
-                print('--module=alsaaudio --device="{}"'.format(device))
+                print(f"--module=alsaaudio --device='{device}'")
```

Comment:
The printout reproduces part of the command line. On IWndows one needs `""` for strings. `''` won't work.
Should look like `print(f'--module=alsaaudio --device="{device}"')`